### PR TITLE
Fix phpcs: env: php\r: No such file or directory

### PR DESCRIPTION
**Pull request by qRoC**

### CODE REVIEW

**Comment by gsherwood** [comment]
What exactly is this change doing?

File: scripts/phpcbf [hunk 1]
```diff
@@ -1,23 +1,46 @@
 #!/usr/bin/env php
+
```
Comment (qRoC) [inline scripts/phpcbf]: fixed, sorry :)

**Comment by aik099** [comment]
Not sure why empty line between shebang and `<?php` does fix it.

Maybe you can post exact error message you're getting? I suppose you've posted only a fragment of it as PR title.

**Comment by aik099** [comment]
So empty line at the end of a file solves that problem? Still not getting how this could fix anything. Maybe that file had mixed line endings in it and by adding/removing line you fixed that?

**Comment by qRoC** [comment]
Just replacing "\r" to "\n" on first line fixes this error.

**Comment by gsherwood** [comment]
I still don't know what error you are trying to solve. Please take some time to write a comment about what this PR is trying to fix and how it is fixing it.

You also mention a `\r` line ending, but these files only have `\n` line endings, which I've just reconfirmed. Plus, this PR doesn't currently change the line ending on the first line anyway.

**Comment by gsherwood** [comment]
Closing as I still don't know what this PR is for.

**Pull request closed without merging**